File: NyanSpecialty.Assistance.Web.Models/CaseDetails.cs

namespace NyanSpecialty.Assistance.Web.Models
{
    public class CaseDetails
    {
        public Case caseDetails { get; set; }
        public List<CaseStatus> caseStatuses { get; set; }
        public List<ServiceProviderAssignment>  serviceProviderAssignment { get; set; }
    }
}

File: NyanSpecialty.Assistance.Web.UI/src/app/models/casedetails.ts
import { Case } from "./case";
import { CaseStatus } from "./casestatus";
import { ServiceProviderAssignment } from "./serviceproviderassignment";

export interface CaseDetails {
    caseDetails: Case,
    caseStatuses: CaseStatus[];
    serviceProviderAssignment: ServiceProviderAssignment[];
    isChecked?: boolean
}
